doc/src/tutorial/intro.rst

```diff
@@ -6,33 +6,33 @@ What is Symbolic Computation?
 =============================
 
 Symbolic computation deals with the computation of mathematical objects
-symbolically.  This means that the mathematical objects are represented
+symbolically. This means that the mathematical objects are represented
```

Comment:
I don't think that these kinds of changes are really needed.

Reply:
oh Sorry, I thought those spaces were there by mistake, my next contribution will be a worthy contribution. I am working on an issue, which is an Easy to fix issue. Thanks.

Reply:
This PR shouldn't stay open forever. Let's either merge or close.
I weakly prefer merge as it does no harm to doc itself (cons: increase repo size, more commits when bisecting).

Reply:
It doesn't do significant harm but it also doesn't really improve anything. Opinions differ on whether there should be one or two spaces after a full stop. Someone could just as easily come along in future and try to change all of these back. As far as I know this doesn't actually affect the built docs. If we want to have a uniform opinion about this then it should be in the style guide:
https://docs.sympy.org/latest/guides/contributing/documentation-style-guide.html

Reply:
Make sense. Then we should probably close it.